cpp/tests/jit/jit-cache-test.hpp

```diff
@@ -45,7 +45,11 @@ struct JitCacheTest : public ::testing::Test, public cudf::jit::cudfJitCache {
   void purgeFileCache()
   {
 #if defined(JITIFY_USE_CACHE)
-    boost::filesystem::remove_all(cudf::jit::getCacheDir());
+    std::vector<boost::filesystem::path> file_paths;
```

Comment:
A quick comment on why it's done this way would be good, so no one accidentally recreates this issue in the future.

Reply:
Makes sense. It's not immediately apparent from the code why the directory isn't deleted.